.gitignore

```diff
@@ -0,0 +1,4 @@
+/bazel-*
+/.ijwb/
+/.idea/
+/grafana/src/
```

Comment:
why?

Reply:
It's a scratch directory for grafanalib. I'll add a comment.